Repo-Integration/Encryption/README.md

```diff
@@ -0,0 +1,39 @@
+## Encrypting Private Registry Credentials for hostRule Configurations
+
+Mend Repository Integration customers often require encrypted credentials to access private registries for package resolution.
+
+Typically, users can encrypt credentials through the following methods:
+
+- **Cloud-Hosted Integrations:** [Mend Repo Integration Encryption page](https://mend-resources.mend.io/index-gh.html)
+- **Self-Hosted Repository Integrations:** Using a page with a public key created by the user
```

Comment:
Add Gitlab, BB, and GHE.  Got you started with GHE
https://docs.mend.io/bundle/integrations/page/configure_mend_for_github_enterprise_to_resolve_your_private_dependencies.html